Tests/Opc.Ua.Server.Tests/MasterNodeManagerTests.cs

```diff
@@ -0,0 +1,269 @@
+/* ========================================================================
+ * Copyright (c) 2005-2020 The OPC Foundation, Inc. All rights reserved.
+ *
+ * OPC Foundation MIT License 1.00
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following
+ * conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * The complete license agreement can be found here:
+ * http://opcfoundation.org/License/MIT/1.00/
+ * ======================================================================*/
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+
+namespace Opc.Ua.Server.Tests
+{
+    /// <summary>
+    /// Test <see cref="MasterNodeManager"/>
+    /// </summary>
+    [TestFixture, Category("MasterNodeManager")]
+    [SetCulture("en-us"), SetUICulture("en-us")]
+    [Parallelizable]
+    public class MasterNodeManagerTests
+    {
+        #region Test Methods
+        /// <summary>
+        /// Test for registering a namespace manager for a namespace
+        /// not contained in the server's namespace table
+        /// </summary>
+        [Test]
+        public async Task RegisterNamespaceManagerNewNamespace()
+        {
+            var fixture = new ServerFixture<StandardServer>();
```

Comment:
I considered moving this out into a single fixture class member and leveraging `OneTimeSetUp`/`OneTimeTearDown`; however, the tests are all very dependent on a clean/known base state and having them own their own fixture should allow for them to be parallelized.

Would be great to get input on the approach here.

Reply:
Ithink you don't have much choice to be in the clean state. And now you can run in parallel, so lgtm!